adagrams/game.py

```diff
@@ -1,11 +1,120 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    ('A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'): 1,
+    ('D', 'G'): 2,
+    ('B', 'C', 'M', 'P'): 3,
+    ('F', 'H', 'V', 'W', 'Y'): 4,
+    ('K',): 5,
+    ('J', 'X'): 8,
+    ('Q', 'Z'): 10
+}
+
+
+def create_letter_pool_list():
+    '''returns a total list of letters from LetterPool'''
+    letter_pool_list = [letter for letter, freq in LETTER_POOL.items() for _ in range(freq)]
```

Comment:
Awesome work on the list comprehension! A warning is just not to make them too complex because then you lose readability.  Also, great work on implementing that weighted distribution. 🎉